### ISSUE TITLE
Баг на странице прогресса перевода

### ISSUE THREAD

**Issue by ReWiG** [LaravelRUS/laravel.su#40]
![изображение](https://user-images.githubusercontent.com/5054805/146184179-e3a343e2-cc51-400b-8b01-de7cc25e1437.png)
Странное поведение сейчас наблюдается на странице https://laravel.su/status/8.x после обновления данных. Коммит один и тот же, но пишет что отстает на 2 коммита. И такая же ситуация у многих других актуальных файлов

**Comment by ReWiG** [comment]
В теории этот баг будет правиться? По идее хорошо бы уже начать работать над переводом документации 8.х

**Comment by slider23** [comment]
Да, надо бы постараться вернуться к нормальной жизни в ближайшее время.

**Comment by SerafimArts** [comment]
Сорь, я что-то совсем закопался в последнее время =( 

Постараюсь на этих выходных заняться. Главное чтоб из головы не вылетело.
